### PR TITLE
Update README.md to match new commands w/o -c

### LINKED ISSUE #100: Modify README.md to reflect v2 release
Change to reflect new command style / structure

cc: @Aerex

### PR DESCRIPTION
**Pull request by maximilien**
Fixes #100 

1. Remove old slack
2. Remove all usage of `-c <command-name>` to `<command-name>
3. Various small changes / corrections
4. Added bash to bash code snippets

cc: @Aerex

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,14 +23,16 @@ Two additional ways, besides Gitter or Slack chat above, to contact us:
 ### Getting Latest Executable: i18n4go
 --------------------------------------
 
-Assuming you have a valid [Golang 1.4.2](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
+Assuming you have a valid [Golang 1.19.5](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
```
Comment (Aerex) [inline README.md:26]: Nit: I think we can lower the min go version. Maybe use 1.17? for the min?

Go modules came to effect as of 1.11 ([blog](https://go.dev/blog/migrating-to-go-modules))

```suggestion
Assuming you have a valid [Golang 1.17](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
```

File: README.md [hunk 2]
```diff
@@ -25,14 +23,16 @@ Two additional ways, besides Gitter or Slack chat above, to contact us:
...
 
 ```
 $ go get github.com/maximilien/i18n4go/i18n4go
```
Comment (Aerex) [inline README.md:29]: Nit: Using `go get` is deprecated for installing binaries.  It is recommend by google to use `go install` to install binaries and compiled packages ([doc](https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies))

Also, as of 1.18 `go get` will not build packages so it is better to use `go install` in the long run. ([blog](https://go.dev/doc/go-get-install-deprecation))

```suggestion
$ go install github.com/maximilien/i18n4go/i18n4go
```

File: README.md [hunk 3]
```diff
@@ -117,18 +117,18 @@ For instance to create `fr_FR` file(s) for French and every other locale_Languag
 Printing the usage help: `$ i18n4go -h` or `$ i18n4go --help`
```
Comment (Aerex) [inline README.md:117]: Typing `i18n4go` prints out the help text too.

```suggestion
Printing the usage help: `$ i18n4go` . 
Printing  usage help for commands: `$ i18n4go <command> --help|-h`

**Review by Aerex — COMMENTED** [review]
Looks good. Add a few comments

**Comment by maximilien** [comment]
OK addressed changes. Let's merge once checks done

**Pull request merged**